src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/ConnectionSetupSubForms/PunchoutCatalogConnectionCartForm.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Form\ConnectionSetupSubForms;
+
+use Generated\Shared\Transfer\PunchoutCatalogConnectionCartTransfer;
+use Spryker\Zed\Kernel\Communication\Form\AbstractType;
+use Symfony\Component\Form\CallbackTransformer;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Range;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\PunchoutCatalogsConfig getConfig()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Communication\PunchoutCatalogsCommunicationFactory getFactory()
+ */
+class PunchoutCatalogConnectionCartForm extends AbstractType
+{
+    protected const FIELD_LABEL_MAX_DESCRIPTION_LENGTH = 'Set Description length on "Transfer to Requisition"';
+    protected const FIELD_LABEL_ENCODING = 'Cart Encoding';
+    protected const FIELD_LABEL_MAPPING = 'Cart Mapping';
+    protected const FIELD_LABEL_DEFAULT_SUPPLIER_ID = 'Default Supplier ID';
+
+    protected const MAX_DESCRIPTION_LENGTH = 99999;
```

Comment:
Why exactly 99999?

Reply:
It came from requirements, see https://spryker.atlassian.net/wiki/spaces/CORE/pages/931495983/PS-8578+Punch+Out
And https://spryker.atlassian.net/browse/PS-9323